protoc-gen-grpc-gateway/internal/gengateway/template.go

```diff
@@ -381,6 +381,8 @@ var filter_{{ .Method.Service.GetName }}_{{ .Method.GetName }}_{{ .Index }} = {{
 			}
 	}
 	{{- end }}
+{{- else }}
+    io.ReadAll(req.Body)
```

Comment:
Instead of this, how about
```suggestion
    io.Copy(io.Discard, req.Body)
```
It might optimize a bit better, and makes it clear that we're just trying to discard it.